### PR TITLE
Masthead images hidden div

### PR DESCRIPTION
**Pull request by Keqi**

### CODE REVIEW

File: src/components/masthead/masthead.hbs [hunk 1]
```diff
@@ -89,5 +90,3 @@
   {{/if}}
 
 </article>
-
```
Comment (mkunkel) [inline src/components/masthead/masthead.hbs]: I understand why one empty line was deleted, but we should have a single blank line at the end of a file.

Comment (Keqi) [inline src/components/masthead/masthead.hbs]: @mkunkel When I looked at the file view in Github, it seems to have one blank line at the end of file.

Comment (Keqi) [inline src/components/masthead/masthead.hbs]: That is weird. My text editor adds one blank line at the end too.

Comment (Keqi) [inline src/components/masthead/masthead.hbs]: I recreated that PR here: https://github.com/lonelyplanet/rizzo-next/pull/621. It does not remove last empty line.

**Pull request closed without merging**